clients/docs/includes/configs/cloud/restproxy.config

```diff
@@ -0,0 +1,10 @@
+# Required connection configs for Kafka producer, consumer, and admin
+bootstrap.servers={{ BROKER_ENDPOINT }}
+security.protocol=SASL_SSL
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+sasl.mechanism=PLAIN
+client.bootstrap.servers={{ BROKER_ENDPOINT }}
+client.security.protocol=SASL_SSL
+client.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+client.ssl.endpoint.identification.algorithm=https
```

Comment:
@cryoshida is this a required config, or is the default already like the Java client and can it be removed?

Reply:
same as above comment.